ccd-adapter/build.gradle

```diff
@@ -22,6 +22,19 @@ dependencies {
 
 }
 
+configurations {
+  testArtifacts.extendsFrom testRuntime
+}
+
+task testJar (type: Jar) {
```

Comment:
Why do you want to test on each deploy ;)

Reply:
I have no idea why this is here, @dharmendrak ?